crypto/cmp/cmp_local.h

```diff
@@ -48,16 +48,22 @@ struct ossl_cmp_ctx_st {
     void *http_cb_arg; /* allows to store optional argument to cb */
 
     /* server authentication */
-    int unprotectedErrors; /* accept neg. response with no/invalid protection */
-                           /* to cope with broken server */
+    /*
+     * unprotectedErrors may be set accept negative response with no protection
+     * of invalid protection in order to cope with broken server
```

Comment:
I don’t really understand this sentence

Reply:
Oh, I made two critical typos when re-working the sentence. Ok now?

Reply:
Yes. At least from the grammar POV ;-)